keep/api/routes/incidents.py

```diff
@@ -708,7 +710,10 @@ def change_incident_status(
 
     # We need to do something only if status really changed
     if not change.status == incident.status:
-        result = change_incident_status_by_id(tenant_id, incident_id, change.status)
+        end_time = (None, datetime.utcnow())[change.status == IncidentStatus.RESOLVED]
```

Comment:
It's hard to read such syntax, plus it includes automatic type casting. Let's stick to old-style 
```suggestion
        end_time = datetime.utcnow() if change.status == IncidentStatus.RESOLVED else None
```